cmd/mimir/help-all.txt.tmpl

```diff
@@ -1639,10 +1639,10 @@ Usage of ./cmd/mimir/mimir:
     	Enable running rule groups against multiple tenants. The tenant IDs involved need to be in the rule group's 'source_tenants' field. If this flag is set to 'false' when there are already created federated rule groups, then these rules groups will be skipped during evaluations.
   -ruler.tenant-shard-size int
     	The tenant's shard size when sharding is used by ruler. Value of 0 disables shuffle sharding for the tenant, and tenant rules will be sharded across all ruler replicas.
-  -runtime-config.file string
-    	File with the configuration that can be updated in runtime.
+  -runtime-config.file value
```

Comment:
Actually this is still a string, but I'm guessing that it puts `value` there because of the `flagext.StringSliceCSV` type, right? 
If it's easy then it would be nice if we could fix that, but otherwise it's not a blocker.